scripts/Elemental_Analysis.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import, print_function
+
+import PyQt4.QtGui as QtGui
+import PyQt4.QtCore as QtCore
+
+import sys
+
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_widget import PeriodicTable
+from Muon.GUI.Common import message_box
+
+
+class ElementalAnalysisGui(QtGui.QMainWindow):
+    def __init__(self, parent=None):
+        super(ElementalAnalysisGui, self).__init__(parent)
+
+        self.ptable = PeriodicTable()
+        self.ptable.register_table_changed(self.table_changed)
```

Comment:
I think these connects can just go into the presenter, along with the slots/functions. Since what will be doing is then calling the model to get the values.